From 89746fb9d788f654a8f95e7f00afaa02cf4d9512 Mon Sep 17 00:00:00 2001
From: Ionut Nechita <ionut_n2001@yahoo.com>
Date: Fri, 22 Apr 2022 13:13:37 +0300
Subject: [PATCH 25/52] SUNLIGHT: block: set rq_affinity to force full
 multithreading I/O requests

Description:
 - force full multithreading I/O requests
   /* force complete on same CPU */

Bug: N/A
Change-Id: Ie9a86fc49161cb669e47bda088d984b48fde4d51
Signed-off-by: Ionut Nechita <ionut_n2001@yahoo.com>
---
 include/linux/blkdev.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/blkdev.h b/include/linux/blkdev.h
index 60d016138997..8f2c19b0871d 100644
--- a/include/linux/blkdev.h
+++ b/include/linux/blkdev.h
@@ -563,6 +563,7 @@ struct request_queue {
 
 #define QUEUE_FLAG_MQ_DEFAULT	((1 << QUEUE_FLAG_IO_STAT) |		\
 				 (1 << QUEUE_FLAG_SAME_COMP) |		\
+				 (1 << QUEUE_FLAG_SAME_FORCE) |		\
 				 (1 << QUEUE_FLAG_NOWAIT))
 
 void blk_queue_flag_set(unsigned int flag, struct request_queue *q);
-- 
2.35.1

